screens/ProductionScreens/BudgetMain.jsx

```diff
@@ -0,0 +1,12 @@
+import { View, Text } from "react-native";
+
+export default function BudgetHomeScreen({ route }) {
```

Comment:
*** 2 cant tell if they're the same thing or one for all budgets and one for a specific budget